config/locales/de.yml

```diff
@@ -169,15 +169,18 @@ de:
         title: Titel
```

Comment:
Please follow [this procedure for new translations](https://github.com/codevise/pageflow/wiki/Adding-new-Translation-Keys).